var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -164,10 +164,36 @@ def install(self, spec, prefix):
 
         install_tree('lib', prefix.lib)
         install_tree('include', prefix.include)
+
+        # Create separate libexec directory tree for libraries and
+        # headers that includes a "fake MPI" header needed for
+        # sequential MUMPS builds (i.e., without MPI) when linking
+        # MUMPS to some libraries (e.g., IPOPT). This "fake MPI"
+        # header will shadow headers in MPI implementations (e.g.,
+        # MPICH, OpenMPI), so there should two separate include
+        # directories: (1) without the "fake MPI" header
```

Comment:
> can you use MUMPS build with +mpi in sequential mode (i don't know)?

I don't know either. It's not important enough for me to try and figure this out. Part of the reason for this pull request is because IPOPT can't be built using a MUMPS build with MPI. IPOPT was not originally developed to use MPI, and they use the fake MPI of MUMPS instead, but the existing MUMPS package in spack does not include the headers for this library. I'll make the change you suggest because this PR needs to land first before I submit a PR with an IPOPT package.